wordle-web/pages/game.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="344">
+    <v-card-item>
+      <v-card-title> Hurdle Over This Wordle!!! </v-card-title>
+      <v-card-subtitle> <strong class="text-red-accent-4">hurdle it. I dare you. </strong> </v-card-subtitle>

```

Comment:
I would make sure the colors that are used match for style in the future. I see you use a red text color and then an orange one for text and then your button is blue. Make sure they all match or have similar hues to make it cohesive!

Reply:
Change it all to red